recipes/openssl/3.x.x/conanfile.py

```diff
@@ -85,11 +85,13 @@ class OpenSSLConan(ConanFile):
         "no_whirlpool": [True, False],
         "no_zlib": [True, False],
         "openssldir": [None, "ANY"],
+        "securitylevel":  [None] + list(range(0, 6)),
```

Comment:
why None?

Reply:
I thought it would be a good idea to have it to keep the default behavior of OpenSSL.
As far as I know, in releases < v.3.2.0 the default value is 1, but starting from v3.2.0 it has changed to 2 ( [https://www.openssl.org/news/cl32.txt](https://www.openssl.org/news/cl32.txt) )

> * The default SSL/TLS security level has been changed from 1 to 2. RSA,
   DSA and DH keys of 1024 bits and above and less than 2048 bits and ECC keys
   of 160 bits and above and less than 224 bits were previously accepted by
   default but are now no longer allowed. By default TLS compression was
   already disabled in previous OpenSSL versions. At security level 2 it cannot
   be enabled.

Reply:
Fair enough